src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -1674,6 +1715,23 @@ PUBLIC_API STATUS signalingClientGetStateString(SIGNALING_CLIENT_STATE, PCHAR*);
  */
 PUBLIC_API STATUS signalingClientDeleteSync(SIGNALING_CLIENT_HANDLE);
 
+/**
+ * @brief Get the relevant/all metrics based on the RTCStatsType field. This does not include
+ * any signaling related metrics
+ *
+ * @param PRtcPeerConnection Peer connection for which the stats need to be collected
+ * @param[in/out] PRtcStats The stats object with the RTCStatsType field populated
+ */
+PUBLIC_API STATUS getStats(PRtcPeerConnection, PRTCStats);
```

Comment:
As we are using C, all the names are in the same namespace increasing a chance of random name collisions with the customers workspace. Please use a little more descriptive name. For example getRtcPeerConnectionStats

Reply:
Done